### PR TITLE
5413 loyalte program section

### PR DESCRIPTION
**Pull request by spavelo**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -83,174 +83,156 @@
...
+                    <div class="table_persent">18,5%</div>
+                    <div class="table_persent">20,2%</div>
+                    <div class="table_persent">19,5%</div>
+                    <div class="table_persent">20,5%</div>
+                    <div></div>
+                </div>
+                <button>порівняти з іншими депозитами</button>
             </div>
-            <div class="info_block">
+            <div class="loialty_program">
```
Comment (vladyslav28) [inline src/index.html]: use .loyalte

File: src/index.html [hunk 2]
```diff
@@ -83,174 +83,156 @@
...
-            <div class="footer_container">
-                <div>
-                    <div class="category small_fill_1">корисні посилання</div>
-                    <button>Відділення та банкомати</button>
-                    <button>Оплата комунальних послуг</button>
-                    <button>Поповнення мобільного</button>
-                    <button>Контакти</button>
-                    <button>Мобільні додатки</button>
+        <div class="open_deposit_way">
+            <div class="content">
```
Comment (vladyslav28) [inline src/index.html]: Why do we need this .content

File: src/index.html [hunk 3]
```diff
@@ -83,174 +83,156 @@
...
-                            <img src="/img/footer/twitter.png" alt="">
-                        </a>
-                        <a href="#">
-                            <img src="/img/footer/youtube.png" alt="">
-                        </a>
-                    </div>
+                <div>За додатковою інформаію можна звертатися до служби підтримки банку онлайн, заповнити
+                    онлайн-заявку нижче, або звертатися за телефоном:
                 </div>
+                <div class="phone_number">
```
Comment (vladyslav28) [inline src/index.html]: div open and close should be in one row as it's small

File: src/index.html [hunk 4]
```diff
@@ -83,174 +83,156 @@
...
-            <div class="category small_fill_1">&copy ОЩАДБАНК, від 1991</div>
-            <div class="supcategory small_fill_1">Ліцензія Національного банку Укаїни №148 від 05.10.2011</div>
+            <div class="map">
+                <img src="/img/content/map.jpg" alt="">
+                <button>мапа усіх відділень</button>
+            </div>
+        </div>
+        <div class="">інформація, що стане в нагоді</div>
+        <div class="">Як відбувається зарахування коштів?</div>
+        <div class="">Готівкою через касу в день відкриття депозиту
```
Comment (vladyslav28) [inline src/index.html]: We don't need empty atrtibutes

Comment (vladyslav28) [reply src/index.html]: Please delete class

Comment (vladyslav28) [reply src/index.html]: around all project

File: src/style/layout/_article.scss [hunk 5]
```diff
@@ -1,7 +1,120 @@
...
+    @extend .x_small_white;
+    font-family: $font_fourth;
+
+    .content_bold {
+      font-weight: 700;
+      padding: 20px 0;
+    }
+
+    .phone_number {
+      padding-top: 20px !important;
```
Comment (vladyslav28) [inline src/style/layout/_article.scss]: Do we really need !important ?

File: src/index.html (hunk 1 above)
Comment (spavelo) [reply src/index.html]: DONE

File: src/index.html (hunk 2 above)
Comment (spavelo) [reply src/index.html]: use for more easier read

File: src/index.html (hunk 3 above)
Comment (spavelo) [reply src/index.html]: DONE

File: src/index.html (hunk 4 above)
Comment (spavelo) [reply src/index.html]: this is draft, and this classes will be use in the near time

**Pull request merged**